### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by mattscepter**
<< Describe the changes >>

Closes:

### CODE REVIEW

File: lib/commons/dom/get-visible-child-text-rects.js [hunk 1]
```diff
@@ -44,6 +51,7 @@ const getVisibleChildTextRects = memoize(
      *
      * @see https://github.com/dequelabs/axe-core/issues/4253
      */
+
```
Comment (sajalg-bstack) [inline lib/commons/dom/get-visible-child-text-rects.js:54]: [nit] Unwanted empty space

File: lib/commons/dom/get-overflow-hidden-ancestors.js [hunk 2]
```diff
@@ -17,8 +18,18 @@ const getOverflowHiddenAncestors = memoize(
 
     const overflow = vNode.getComputedStylePropertyValue('overflow');
 
-    if (overflow === 'hidden') {
-      ancestors.push(vNode);
+    if (cache.get('200%ZoomRule')) {
```
Comment (sajalg-bstack) [inline lib/commons/dom/get-overflow-hidden-ancestors.js:21]: cache.set is present in another PR?

File: lib/commons/dom/create-grid.js [hunk 3]
```diff
@@ -27,6 +27,9 @@ export default function createGrid(
 ) {
   // Prevent multiple calls per run
   if (cache.get('gridCreated') && !parentVNode) {
+    if (cache.get('gridSize')) {
+      return cache.get('gridSize');
+    }
```
Comment (Abhi3685) [inline lib/commons/dom/create-grid.js:32]: where is this property getting set?

File: lib/commons/dom/get-element-stack.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ import createGrid from './create-grid';
  * @param {Node} node
  * @return {Node[]}
  */
-function getElementStack(node) {
+function getElementStack(node, isCoordsPassed = false, x, y) {
```
Comment (Abhi3685) [inline lib/commons/dom/get-element-stack.js:12]: default arguments should be at the end - x & y should also be defaults?

File: lib/commons/dom/get-rect-stack.js [hunk 5]
```diff
@@ -1,12 +1,25 @@
 import visuallySort from './visually-sort';
 import { getRectCenter } from '../math';
 
-export function getRectStack(grid, rect, recursed = false) {
+export function getRectStack(
+  grid,
+  rect,
+  recursed = false,
```
Comment (Abhi3685) [inline lib/commons/dom/get-rect-stack.js:7]: default arguments should be at the end - there must not be any mandatory arguments after default arguments

**Review by Abhi3685 — CHANGES_REQUESTED** [review]

**Review by Abhi3685 — DISMISSED** [review]

**Pull request merged**